### PR TITLE
Mirror changes in channel topic from IRC to Slack.

### PR DESCRIPTION
**Pull request by redplanks**
Solves part of #7

### CODE REVIEW

**Comment by kkuehler** [comment]
Looks good

File: slackbridge/bots.py [hunk 1]
```diff
@@ -101,6 +103,16 @@ def check_slack_rtm(self):
             channel = self.channels[message['channel']]
             return user_bot.post_to_irc('#' + channel['name'], message['text'])
 
+    def topicUpdated(self, user, channel, newTopic):
```
Comment (jvperrin) [inline slackbridge/bots.py]: I'd name this something like `update_topic` instead to stay in the same style with the rest of the methods. Same with `newTopic` -> `new_topic`.

Comment (redplanks) [reply slackbridge/bots.py]: This is an implementation of an event handler in IRCClient that gets called whenever the topic changes or a channel is joined for the first time, so I don't think it's possible to change the name of the function. I can change the topic variable name though.

Comment (jvperrin) [reply slackbridge/bots.py]: Oh, that's awkward, I forgot that's how it works. Maybe put a comment saying that it gets called by twisted when the topic is updated so it's clearer?

**Pull request merged**